.travis.yml

```diff
@@ -1,12 +1,11 @@
-language: c
-
 services:
   - docker
 
 script:
-  - docker run -w /repo -v $(pwd):/repo --entrypoint "/bin/bash" -it ocaml/opam2:alpine-3.9-ocaml-4.07 ".travis-ci.sh"
+#  - docker pull marionebl/ocaml-exercism-ci
+#  - docker build -t marionebl/ocaml-exercism-ci -f .travis/Dockerfile .
+  - docker run -w /repo -v $(pwd):/repo -it marionebl/ocaml-exercism-ci .travis/.travis-ci.sh
```

Comment:
This brings down build time considerably, requires a docker hub repo though. Moving the prebuilt image to e.g. exercism/ocaml-ci might be good idea?